srcpkgs/qtile/template

```diff
@@ -18,3 +20,11 @@ post_install() {
 	vinstall resources/qtile.desktop 644 usr/share/xsessions
 	vlicense LICENSE
 }
+
+qtile-wayland_package() {
+	depends="${sourcepkg}>=${version}_${revision}
+	 python3-pywlroots-${_wlroots} python3-pywayland python3-xkbcommon"
```

Comment:
are pywaland and xkbcommon necessary to list here? they're already dependencies of pywlroots